nautobot/docs/user-guide/administration/tools/nautobot-server.md

```diff
@@ -542,11 +542,8 @@ This will run the following management commands with default settings, in order:
 --- 2.0.0
     With the removal of `django-cacheops` from Nautobot, this command no longer runs `invalidate all`.
 
-+++ 2.0.0
-    Added [`build_ui`](#build_ui) to this command's default behavior.
```

Comment:
Instead of removing this historical note, we should add a `+/- 2.0.3` admonition documenting the change in behavior.